app/assets/stylesheets/core/_settings.scss

```diff
@@ -1 +1,3 @@
 @import "foundation_and_overrides";
+@import "foundation";
+@include foundation-everything;
```

Comment:
@timurvafin  `@include foundation-everything` that include all components from foundation 6 it isn't file. Each component has an [export mixin](http://foundation.zurb.com/sites/docs/sass.html#adjusting-css-output).